src/decNumber/decBasic.c

```diff
@@ -113,7 +113,7 @@ static decFloat * decCanonical(decFloat *result, const decFloat *df) {
   #endif
   } // block
 
-  // Loop to repair a non-canonical coefficent, as needed

```

Comment:
This is a third-party library, so please drop the changes, and report to the upstream if you can.

Reply:
Ah, I didn't realise. I'm used to seeing copied dependencies in "vendor" or "third_party" folders.